src/braille/internal/braillewriter.h

```diff
@@ -20,12 +20,12 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-#ifndef MU_IMPORTEXPORT_BRAILLEWRITER_H
-#define MU_IMPORTEXPORT_BRAILLEWRITER_H
+#ifndef MU_BRAILLE_BRAILLEWRITER_H
+#define MU_BRAILLE_BRAILLEWRITER_H
 
 #include "project/inotationwriter.h"
 
-namespace mu::iex::braille {
+namespace mu::engraving {
```

Comment:
Excuse me if I have missed previous discussion about this, but why is it in namespace `mu::engraving` and not in `mu::braille`?

Reply:
We haven't discussed it, but my reason for doing this was because there are lots of terms from libmscore used in these files, particularly in `src/braille/internal/braille.h`, so it saves seeing `engraving::` everywhere.

These files are internal to the module, so it doesn't really matter which namespace they use. The public module classes do use the `braille` namespace.

It's also done this way in the MusicXML export module, for example.

Reply:
Okay. I think it would be good to clean this up at some point, also in the MusicXML module, but we can do that later, in order not to block this PR.